cmd/awscollector/main.go

```diff
@@ -17,6 +17,7 @@ package main // import "aws-observability.io/collector/cmd/awscollector"
 
 import (
 	"fmt"
+	"github.com/aws-observability/aws-otel-collector/pkg/defaultcomponents"
```

Comment:
This should go back where it came from.  Imports should generally be in three groups: stdlib, other sources, packages from this repo.

Reply:
https://github.com/aws-observability/aws-otel-collector/pull/1911#discussion_r1139301364